tests/console/bind.pm

```diff
@@ -0,0 +1,76 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: bind upstream testsuite
+#          prepare, build, fix broken tests and execute testsuite
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base 'consoletest';
+use testapi;
+use strict;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    if (is_sle('=12-SP3')) {
+        # SLE 12-SP3 neeed gpg-offline to build
+        assert_script_run 'wget http://download.suse.de/ibs/SUSE:/SLE-12:/GA/standard/noarch/gpg-offline-0.1-10.4.noarch.rpm';
+        assert_script_run 'rpm -iv gpg-offline*';
+        # perl-IO-Socket-INET6 for reclimit
+        zypper_call 'in bind rpm-build perl-IO-Socket-INET6';
+    }
+    elsif (is_sle('>=15')) {
+        # dnspython for chain test
+        assert_script_run 'curl -L http://download.suse.de/install/SLP/SLE-15-Module-Public-Cloud-GM/x86_64/DVD1/noarch/python3-dnspython-1.15.0-1.25.noarch.rpm -o python3-dnspython.rpm';
```

Comment:
Maybe it worth extracting this to separate method, which get url and installs rpm from it. Not sure why we use wget in SLE12 and curl for SLE15? Is there a reason for that? AFAIK one of them follows redirects by default.

Reply:
on SLE15 is no wget by default

Reply:
Good to know, thanks!